db/memtable.cc

```diff
@@ -475,7 +475,7 @@ void MemTable::Add(SequenceNumber s, ValueType type,
     }
 
     // The first sequence number inserted into the memtable
-    assert(first_seqno_ == 0 || s > first_seqno_);
+    assert(first_seqno_ == 0 || s >= first_seqno_);
```

Comment:
This is because we now insert multiple key/value with the same sequence number.  So for the first batch we could have multiple insertion all with the same sequence number of the first one, i.e., first_seqno_